### PR TITLE
fixed old and new classifiers regex

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/25204

## Description
Fixed an issue where new and old classifiers where mixed on validate command by editing their regex.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/tests/constants_regex_test.py [hunk 1]
```diff
@@ -28,6 +29,51 @@ def verify(acceptable, unacceptable, matched_regex):
         assert not checked_type(test_path, compared_regexes=matched_regex)
 
 
+def get_test_new_classifiers_paths(folder):
+    acceptable_code_files = {
```
Comment (roysagi) [inline demisto_sdk/commands/common/tests/constants_regex_test.py:33]:
```suggestion
    acceptable_classifiers_paths = {
```

same change in all tests

File: demisto_sdk/commands/common/tests/constants_regex_test.py [hunk 2]
```diff
@@ -84,6 +130,36 @@ def test_script_code_files():
     )
 
 
+def test_new_classifier_files():
+    acceptable_script_code_files, unacceptable_script_code_files = get_test_new_classifiers_paths(CLASSIFIERS_DIR)
```
Comment (roysagi) [inline demisto_sdk/commands/common/tests/constants_regex_test.py:134]:
```suggestion
    acceptable_classifiers_files, unacceptable_classifiers_files = get_test_new_classifiers_paths(CLASSIFIERS_DIR)
```

same change in all tests

File: demisto_sdk/commands/common/tests/constants_regex_test.py (hunk 1 above)
Comment (barchen1) [reply demisto_sdk/commands/common/tests/constants_regex_test.py:33]: done

File: demisto_sdk/commands/common/tests/constants_regex_test.py (hunk 2 above)
Comment (barchen1) [reply demisto_sdk/commands/common/tests/constants_regex_test.py:134]: done

**Pull request merged**